core/xml.js

```diff
@@ -50,6 +51,23 @@ Blockly.Xml.workspaceToDom = function(workspace, opt_noId) {
 };
 
 /**
+ * Encode a list of variables as XML.
+ * @param {!Array.<Blockly.VariableModel>} variableList List of all variable
+ *     models.
+ * @return {!Element} XML document.
```

Comment:
List of XML elements.

WorkspaceToDom returns an XML document because its root tag is `<xml>`, but this function doesn't.